### DISCUSSION TITLE
Client Certificate is not getting prompt from browser

### DISCUSSION THREAD

**Discussion by adsouza75** [nginx/kubernetes-ingress#5443] [Q&A] [answered]
Iam not getting prompt for client certificate from browser 

here is my ingress config

apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  annotations:
    nginx.ingress.kubernetes.io/auth-tls-pass-certificate-to-upstream: "true"
    nginx.ingress.kubernetes.io/auth-tls-secret: default/casecret
    nginx.ingress.kubernetes.io/auth-tls-verify-client: "on"
    nginx.ingress.kubernetes.io/auth-tls-verify-depth: "1"
    nginx.ingress.kubernetes.io/server-snippet: ssl_verify_client optional_no_ca ;
  creationTimestamp: "2024-04-24T13:38:18Z"
  generation: 4
  name: test-localhost
  namespace: default
  resourceVersion: "187097603"
  uid: 52b29924-b54c-41d9-a6c7-cafb6326e32c
spec:
  ingressClassName: nginx-mtls
  rules:
  - host: test.auth-mtls.wealthcare.com
    http:
      paths:
      - backend:
          service:
            name: test
            port:
              number: 80
        path: /
        pathType: Prefix
  tls:
  - hosts:
    - test.auth-mtls.wealthcare.com
    secretName: auth-mtls.wealthcare.com

**Comment by jasonwilliams14** [comment, marked as answer]
@adsouza75 Hello there
The annotations are you using are for a different NGINX Ingress controller project.
This repo is the official NGINX repository for NGINX Ingress controller.
you can see our documentation located at this link, with our annotations for our project. (Note...the other projects annotations will not work with our NGINX Ingress controller project.)

https://docs.nginx.com/nginx-ingress-controller/

https://docs.nginx.com/nginx-ingress-controller/configuration/ingress-resources/advanced-configuration-with-annotations/

Hope this helps.

**Comment by jjngx** [comment]
Answered a year ago, closing.
